tests/autologging/fixtures.py

```diff
@@ -1,10 +1,28 @@
 import os
+import sys
 
 import pytest
 
+import mlflow.utils.logging_utils as logging_utils
 from mlflow.utils.autologging import _is_testing, _AUTOLOGGING_TEST_MODE_ENV_VAR
 
 
+PATCH_DESTINATION_FN_DEFAULT_RESULT = "original_result"
+
+
+@pytest.fixture
+def patch_destination():
```

Comment:
Moved from `tests/autologging/test_autologging_safety_unit.py` for reuse across multiple test suites